db-event-log/src/main/scala/ch/datascience/dbeventlog/commands/EventLogFetch.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ch.datascience.dbeventlog.commands
+
+import java.time.temporal.ChronoUnit.MINUTES
+import java.time.{Duration, Instant}
+
+import cats.MonadError
+import cats.data.NonEmptyList
+import cats.effect.{ContextShift, IO}
+import cats.free.Free
+import cats.implicits._
+import ch.datascience.db.TransactorProvider
+import ch.datascience.dbeventlog.EventStatus._
+import ch.datascience.dbeventlog.{EventBody, EventStatus, IOTransactorProvider}
+import ch.datascience.graph.model.events.CommitId
+import doobie.free.connection.ConnectionOp
+import doobie.implicits._
+import doobie.util.fragments._
+
+import scala.language.higherKinds
+
+private object EventLogFetch {
+  val MaxProcessingTime: Duration = Duration.of(10, MINUTES)
```

Comment:
Maybe we should put this to the environment variable.

Reply:
I was thinking about that but it's so deeply internal :), if you know what I mean, that probably will never be changed. I didn't want to hardcode that value somewhere between the lines and that's why I took it out to the constant.